tests/test_operation_logs.py

```diff
@@ -0,0 +1,31 @@
+
+from tests import factories
+from tests.common import ApiBaseTest
+from webservices.rest import api
+from webservices.resources.operations_log import OperationsLogView
+
+class TestOperationsLog(ApiBaseTest):
+
+    def setUp(self):
+        super().setUp()
+        factories.OperationsLogFactory(candidate_committee_id='00', report_year=2000, status_num=1),
+        factories.OperationsLogFactory(candidate_committee_id='01', report_year=2012, status_num=0),
+        factories.OperationsLogFactory(candidate_committee_id='02', report_year=2014, status_num=1),
+        factories.OperationsLogFactory(candidate_committee_id='03', report_year=2016, status_num=0),
+        factories.OperationsLogFactory(candidate_committee_id='04', report_year=2018, status_num=1),
+        factories.OperationsLogFactory(candidate_committee_id='05', report_year=2020, status_num=1),
+
+    def test_empty_query(self):
+
+        results = self._results(api.url_for(OperationsLogView, candidate_committee_id='10', report_year=2030))
+        self.assertEqual(len(results), 0)
+
+    def test_search_cand_cmte_id(self):
+
+        response = self.app.get(api.url_for(OperationsLogView, candidate_committee_id='01', report_year=2012))
+        self.assertEquals(response.status_code, 200)
+
+    def test_search_cand_cmte_id_with_status_verified(self):
+
+        results = self._results(api.url_for(OperationsLogView, status_num=1))
```

Comment:
@pkfec thanks for adding another test! Because we're testing that `status_num=1` maybe we could `assert` that all the results are `status_num=1`  rather than there being 4 results, in case we add more test data in the future. Something like this, but I'd test it out to be sure :)
```
for result in results:
    self.assert(result.get('status_num') == 1 )
```

Reply:
results object only yields the actual count of the records found in test data.  From response object we can decode the data.  I couldnt find a way to compare the response object against the status_num=1.

Reply:
This test is giving the misleading impression that I could query the endpoint by `status_num`, ie, `api.url_for(OperationsLogView, status_num=0)`, `api.url_for(OperationsLogView, status_num=1)`, etc. Is that correct? Aren't we fixing the query to use `status_num == 1` in https://github.com/fecgov/openFEC/pull/3203/files#diff-362d4bedb02a4abf10fc6a6868a7b021R39?